src/watchmaker/workers/salt.py

```diff
@@ -241,7 +240,9 @@ def _get_formulas_conf(self):
         ]
 
     def _build_salt_formula(self, extract_dir):
-        if self.salt_content:
+
```

Comment:
The spaces around this `if` seem extraneous, and throw off the diff. Let's minimize the changes please